File: 1-TwoSum/1-TwoSum.py
# Last updated: 6/22/2025, 11:05:54 AM
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n=len(nums)
        for i in range(n-1):
            for j in range(i+1,n):
                if nums[i]+nums[j]==target:
                    return [i,j]
        return []

File: 1-TwoSum/1-TwoSum.java
// Last updated: 6/22/2025, 11:05:53 AM
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> res=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            int sub=target-nums[i];
            if(res.containsKey(sub)){
                return new int[]{res.get(sub),i};
            }
            res.put(nums[i],i);
        }
        return new int[]{-1,-1};
    }
}